### PR TITLE
add Plättchen setzen

### PR DESCRIPTION
**Pull request by MarcRaeder**
user can place his symbol

### CODE REVIEW

File: Board.py [hunk 1]
```diff
@@ -1,30 +1,87 @@
...
             for element in line:
-                print(f"|{element}", end='')
+                print(f"|{element}", end="")
             print("|")
+
+    def IsValidTurn(self, element: int) -> bool:
+        element -= 1
+        for line in range(len(self.board)):
+            if self.board[line][element] != self.placeHolder:
+                if self.board[line - 1][element] == self.placeHolder:
```
Comment (DevBastiS) [inline Board.py]: Move conditions to variables

File: Board.py [hunk 2]
```diff
@@ -1,30 +1,87 @@
...
+                    return True
+        return False
+
+    def AddCoinToBoard(self, element: int, playerSymbol: str):
+        element -= 1
+        for line in range(len(self.board)):
+            if self.board[line][element] != self.placeHolder:
+                if self.board[line - 1][element] == self.placeHolder:
```
Comment (DevBastiS) [inline Board.py]: Move conditions to variables

File: MainGame.py [hunk 3]
```diff
@@ -1,55 +1,73 @@
...
+                firstLine: int = 1
+                lastLine: int = 7
 
-                return rowInput
+                inputIsValid = firstLine <= rowInput <= lastLine
+                if inputIsValid:
+                    return rowInput
+                else:
+                    print(
+                        f"Deine Eingabe '{rowInput}' ist ungültig. Versuch es nochmal"
```
Comment (DevBastiS) [inline MainGame.py:70]: Sometimes you move the string for print in the next line sometimes not.

**Review by DevBastiS — CHANGES_REQUESTED** [review]

File: MainGame.py (hunk 3 above)
Comment (MarcRaeder) [reply MainGame.py:70]: yes, because the formatter (black) wants it that way

File: MainGame.py [hunk 4]
```diff
@@ -1,55 +1,73 @@
...
         while True:
-            userName: str = input(f"{greeting}: Please enter your name: ")
+            row: int = self.GetRowInput(player.name)
+
+            isvalidTurn: bool = self.board.IsValidTurn(row)
+
+            if isvalidTurn:
+                self.board.AddCoinToBoard(row, player.symbol)
+                return
+            else:
```
Comment (DevBastiS) [inline MainGame.py]: This else keyword is unnecessary since the return would stop the function.

**Review by DevBastiS — APPROVED** [review]

**Review by stefanluth — APPROVED** [review]
Remove `__pycache__`

**Pull request merged**